README.md

```diff
@@ -1,126 +1,114 @@
 
-3. Include any other integrations you want the agent to use, for example:
+4. (Optional) Include integrations to use with the Agent, for example:
 
-    ```
-    include 'datadog_agent::integrations::mongo'
-    ```
+   ```conf
+   include 'datadog_agent::integrations::mongo'
+   ```
 
-    Some integrations do not come as a dedicated class. To install one of them, add its configuration in the manifest. Below is an example for the `ntp` check:
+    Add the integration configuration to the manifest if it does not have a [manifest with a dedicated class][6]. Below is an example for the `ntp` check:
 
-    ```
-    class { 'datadog_agent':
-        api_key      => "<YOUR_DD_API_KEY>",
-        integrations => {
-            "ntp" => {
-                init_config => {},
-                instances => [{
-                    offset_threshold => 30,
-                }],
-            },
-        },
-    }
-    ```
+   ```conf
+   class { 'datadog_agent':
+       api_key      => "<YOUR_DD_API_KEY>",
+       integrations => {
+           "ntp" => {
+               init_config => {},
+               instances => [{
+                   offset_threshold => 30,
+               }],
+           },
+       },
+   }
+   ```
 
-Installing and pinning specific versions of integrations
---------------------------------------------------------
+#### Integration versions
 
-You can specify a given integration and version number to be installed by using `datadog_agent::install_integration`. This will use the `datadog-agent integration` command to ensure a specific integration is installed or uninstalled.
+To install and pin specific integration versions, specify an integration and version number by using `datadog_agent::install_integration`. This uses the `datadog-agent integration` command to ensure a specific integration is installed or uninstalled, for example:
 
-```
+```conf
 datadog_agent::install_integration { "mongo-1.9":
     ensure => present,
     integration_name => 'datadog-mongo',
     version => '1.9.0',
 }
 ```
 
-The field `ensure` can be either `present` (default) or `absent`, the later being useful to remove a previously pinned version of an integration.
+`ensure` has two options:
+
+- `present` (default)
+- `absent` (removes a previously pinned version of an integration)
 
+### Reporting
 
-Reporting
----------
-Ensure the `dogapi` gem is available on your system as explained earlier.
+Ensure the [dogapi][3] gem is available on your system.
 
-To enable reporting of changes to the Datadog timeline, enable the report processor on your Puppet master, and enable reporting for your clients. The clients send a run report after each check-in back to the master.
+To report changes to your Datadog timeline, enable the report processor on your Puppet master and reporting for your clients. The clients send a run report after each check-in back to the master.
```

Comment:
```suggestion
To enable reporting of changes to your Datadog timeline, enable the report processor on your Puppet master and reporting for your clients. The clients send a run report after each check-in back to the master.
```
I like the previous wording better, since we are not talking about generic reports but the 'reporting' feature in Puppet (https://puppet.com/docs/puppet/latest/reporting_about.html)